.github/workflows/ci.yml

```diff
@@ -34,8 +38,8 @@ jobs:
       - name: Build dist
         run: npm run build
 
-      - name: Run tests
-        run: npm test
+      - name: Run tests (${{ matrix.thread }}-threaded)
+        run: npm run test:${{ matrix.thread }}
```

Comment:
this was more of an experiment — seems like a lot of unnecessary build overhead with basically zero time savings — feel free to revert!

Reply:
update: reverted in https://github.com/readmeio/rdme/pull/870/commits/a870661f83ad789e4d956728019a18dd743f2622